From 40e49bb53218a986b4d6569f3fc2730cc4b71472 Mon Sep 17 00:00:00 2001
From: Michael Catanzaro <mcatanzaro@redhat.com>
Date: Tue, 12 Jul 2022 16:56:56 -0700
Subject: [PATCH] Fix cloop build after 252338@main
 https://bugs.webkit.org/show_bug.cgi?id=242579

Reviewed by Mark Lam.

* Source/JavaScriptCore/CMakeLists.txt:

Canonical link: https://commits.webkit.org/252400@main
---
 Source/JavaScriptCore/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Source/JavaScriptCore/CMakeLists.txt b/Source/JavaScriptCore/CMakeLists.txt
index aef0678e66f3..f4ba2716faef 100644
--- a/Source/JavaScriptCore/CMakeLists.txt
+++ b/Source/JavaScriptCore/CMakeLists.txt
@@ -837,6 +837,7 @@ set(JavaScriptCore_PRIVATE_FRAMEWORK_HEADERS
     inspector/remote/RemoteInspector.h
 
     interpreter/AbstractPC.h
+    interpreter/CLoopStack.h
     interpreter/CallFrame.h
     interpreter/CallFrameInlines.h
     interpreter/CalleeBits.h
